draft-packages/form/KaizenDraft/Form/Primitives/ToggleSwitch/ToggleSwitch.module.scss

```diff
@@ -57,10 +57,20 @@ $focus-ring-offset: 1px;
   transition: left $animation-timing, right $animation-timing;
 }
 
+.checkIcon {
+  color: $color-green-500;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  margin: -10px 0 0 -9px;
```

Comment:
Can we use `text-align: center` or flex to center the icon?

Reply:
There might be a different way, but `text-align` doesn't work. With the icon component within the "dot" and also the check mark icons "visual center" not being its actual center, the positioning is a bit finicky.